website/docs/configuration-options.md

```diff
@@ -1080,6 +1080,8 @@ Use this field to suppress various types of warnings and other notifications fro
 
 The above config will suppress the comment which is added to a PR whenever you close a PR unmerged.
 
+## swift
```

Comment:
Do you think there's anything about Swift support that should be described? e.g. our handling of `from:` etc?